pkgs/tools/misc/mongodb-tools/default.nix

```diff
@@ -22,28 +25,23 @@ buildGoPackage rec {
   };
 
   goDeps = ./deps.nix;
-  
-  buildInputs = [ pkgconfig openssl_1_0_2 libpcap ];
-  
- 
-  buildPhase = ''
-    ./go/src/github.com/mongodb/mongo-tools/build.sh ssl
-  '';
-  
-  buildFlags = [ "-tags ssl" ];
-  
-  allowGoReference = true;
-  
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ openssl_1_0_2 libpcap ];
 
   # Mongodb incorrectly names all of their binaries main
   # Let's work around this with our own installer
-  preInstall = ''
-    mkdir -p $bin/bin
-  '' + toString (map (t: ''
-      go install $goPackagePath/${t}/main
-      mv go/bin/main $bin/bin/${t}
-  ''
-  ) tools) + ''  
-    rm -r go/bin
+  buildPhase = ''
+    runHook preBuild
+    ${stdenv.lib.concatMapStrings (t: ''
+      go build -o "$bin/bin/${t}" -tags ssl -ldflags "-s -w" $goPackagePath/${t}/main
```

Comment:
Go binaries where not stripped correctly.

Reply:
AFAIK Go packages do not refer to the Go compiler even with symbols and debug info. I have just built this package without `-ldflags "-s -w"`.

Reply:
It definitely was referring to go before that commit. I saw it with `strings`. I don't where this was coming from. Nevertheless stripping binaries does not hurt as it results in smaller binaries.